src/sass/feature/_base.scss

```diff
@@ -4,50 +4,56 @@
 @use 'core' as *;
 
 .#{$prefix}-feature {
+  display: flex;
+  flex-direction: column;
+
+  &--horizontal {
+    flex-direction: row;
+
+    .#{$prefix}-feature__image {
+      margin: $spacing-sm $spacing-sm 0 0;
+    }
+  }
+
   &__image img {
     display: block;
     width: 100%;
   }
 
   &__body {
-    margin-top: $spacing-md;
+    > * {
+      margin: 0;
+    }
```

Comment:
I talked to Levi a little about this, because the first item in each variant had a `margin-top`. To refactor this a bit, I was thinking we could do something like this:

```
    * + * {
      margin-bottom: 0;
      margin-top: $spacing-sm;
    }
```

and then remove the `margin-top` from the `&__title` and `&__detail`. That way the first item doesn't have a `margin-top`, but the following items do.